build.cake

```diff
@@ -412,108 +413,44 @@ Task("PackageChocolatey")
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit3-console.exe", Target="tools" },
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit3-console.exe.config", Target="tools" },
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit.engine.api.dll", Target="tools" },
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit.engine.api.xml", Target="tools" },
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit.engine.dll", Target="tools" },
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "Mono.Cecil.dll", Target="tools" }
-		};
-
-		// Used for the creation of the package. Duplicates what is in the extensions
-		// structure, unfortunately. Should be changed.
-		var extensionContent = new []
-		{
-			new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit.engine.addins", Target="tools" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.VSProjectLoader/tools/vs-project-loader.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.NUnitProjectLoader/tools/nunit-project-loader.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.NUnitV2ResultWriter/tools/nunit-v2-result-writer.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.NUnitV2Driver/tools/nunit.v2.driver.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.NUnitV2Driver/tools/nunit.core.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.NUnitV2Driver/tools/nunit.core.interfaces.dll", Target="tools/addins" },
-			new ChocolateyNuSpecContent { Source = TOOLS_DIR + "NUnit.Extension.TeamCityEventListener/tools/teamcity-event-listener.dll", Target="tools/addins" }
-		};
-
-		ChocolateyPack("choco/nunit-console-runner.nuspec",
+		
+		ChocolateyPack("choco/nunit-console-runner.nuspec", 
 			new ChocolateyPackSettings()
 			{
 				Version = packageVersion,
 				OutputDirectory = PACKAGE_DIR,
-				Files = basicContent
+				Files = new [] {
+                    new ChocolateyNuSpecContent { Source = PROJECT_DIR + "LICENSE.txt", Target = "tools" },
+                    new ChocolateyNuSpecContent { Source = PROJECT_DIR + "NOTICES.txt", Target = "tools" },
+                    new ChocolateyNuSpecContent { Source = PROJECT_DIR + "CHANGES.txt", Target = "tools" },
+			        new ChocolateyNuSpecContent { Source = CHOCO_DIR + "VERIFICATION.txt", Target = "tools" },
+			        new ChocolateyNuSpecContent { Source = CHOCO_DIR + "nunit.choco.addins", Target = "tools" },
+                    new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit-agent.exe", Target="tools" },
+                    new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit-agent.exe.config", Target="tools" },
+                    new ChocolateyNuSpecContent { Source = CHOCO_DIR + "nunit-agent.exe.ignore", Target="tools" },
```

Comment:
The prevent chocolatey from creating automatic shims on the path for running those files. This way, nunit3.console.exe works as if it were on the path, but the agent files don't. I did that to prevent folks from running them directly as has occasionally happened in the past.